META-INF/plugin.xml

```diff
@@ -1,7 +1,7 @@
 <idea-plugin version="2">
   <id>com.chrisfolger.needsmoredojo</id>
   <name>Needs More Dojo</name>
-  <version>0.7</version>
+  <version>0.8.1</version>
```

Comment:
https://github.com/cefolger/needsmoredojo/pull/181 suggests moving the file to __resources/META-INF/plugin.xml__ and updating __resources/META-INF/MANIFEST.MF__.

Reply:
I don't think this file location particularly matters, but if this becomes a problem I'll happily move it.